opm/parser/eclipse/EclipseState/Grid/EclipseGrid.hpp

```diff
@@ -55,6 +55,7 @@ namespace Opm {
     public:
         explicit EclipseGrid(const std::string& filename);
         explicit EclipseGrid(const EclipseGrid& srcGrid);
+        explicit EclipseGrid(const EclipseGrid& src, const std::vector<double>& zcorn , const std::vector<int>& actnum);
```

Comment:
Not that it matters, but `explicit` does nothing in non-1-arity constructors, since the possibility for implicit conversion just isn't there due to the arity.